sycl/include/CL/sycl/detail/queue_impl.hpp

```diff
@@ -120,11 +121,11 @@ class queue_impl {
     m_Exceptions.clear();
   }
 
-  cl_command_queue createQueue() const {
+  cl_command_queue createQueue() {
     cl_command_queue_properties CreationFlags = 0;
 
-    // FPGA RT can't handle out of order queue - create in order queue instead
-    if (!m_Device.is_accelerator()) {
+    // If create out-of-order queue
```

Comment:
I think new version of `if` statement below doesn't need this comment.

Reply:
removed this comments